components/compositing/compositor.rs

```diff
@@ -128,8 +131,10 @@ pub struct IOCompositor<Window: WindowMethods + ?Sized> {
     /// The port on which we receive messages.
     port: CompositorReceiver,
 
-    /// The root pipeline.
-    root_pipeline: RootPipeline,
+    /// The root content pipeline ie the pipeline which contains the main frame
+    /// to display. In the WebRender scene, this will be the one child of another
```

Comment:
Is this supposed to read ` the *only* child of` instead of  `the one child of`

Reply:
I've updated this comment.